src/views/clusteroverview/overview/components/details-card/utils/models.ts

```diff
@@ -0,0 +1,69 @@
+import { K8sModel } from '@openshift-console/dynamic-plugin-sdk';
```

Comment:
if the models in here can be used by others we should consider moving this file to src/utils/models at some point :)

Reply:
I plan on adding them to the `kubevirt-api` repo and then deleting them here.

Reply:
Migration to `kubevirt-api` finished and references updated.